pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -677,7 +677,7 @@ func (nc *Controller) monitorNodeHealth() error {
 			}
 			return false, nil
 		}); err != nil {
-			klog.Errorf("Update health of Node '%v' from Controller error: %v. "+
+			klog.Errorf("Update health of Node %s from Controller error: %v. "+
```

Comment:
should be `%q`

Reply:
All other log infos use %s or %v to format node.name in this file.